sbc_compassion/models/import_letters_history.py

```diff
@@ -214,7 +137,6 @@ def button_save(self):
 
     @api.multi
     def button_review(self):
-        """ Returns a form view for import lines in order to browse them """
```

Comment:
I don't think it's an improvement to remove all the comments from the code. It's ok when the function name is self explanatory and the code easy to understand but in general commenting the code is a good practice.